### PR TITLE
Remove dependency on Exif parsing library.

### PR DESCRIPTION
**Pull request by dbrant**
More unnecessary libraries that can be removed with just a little restructuring.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java [hunk 1]
```diff
@@ -17,37 +9,38 @@
...
-        }
-        /**
-         * We parse through all the tags in the IPTC directory  if the tagname equals "Special Instructions".
-         * And the description string starts with FBMD.
-         * Then the source of image is facebook
-         * */
-        for (Tag tag : iptcDirectory.getTags()) {
-            if (tag.getTagName().equals("Special Instructions") && tag.getDescription().substring(0, 4).equals("FBMD")) {
+            if (psBlockOffset > 0 && fbmdOffset > 0
+                    && fbmdOffset > psBlockOffset && fbmdOffset - psBlockOffset < 0x80) {
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java]: While removing a 656 KB library is great, in my humble opinion I would say that this implementation is much less readable, replacing easy-to-read business logic with byte manipulation and unexplained magic codes such as "8BIM" and 0x80.

Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java]: Maybe some comments/explanations could help. In that case would you still think this is less readable @nicolas-raoul ?

Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/upload/ReadFBMD.java]: @dbrant is it possible to add some comments here to make it more readable?

**Comment by maskaravivek** [comment]
The build is failing because of the following error which has already been fixed on master. So going ahead and merging it. Confirmed with @ashishkumar468 

```
> Task :app:mergeBetaDebugResources FAILED
/home/travis/build/commons-app/apps-android-commons/app/src/main/res/values-en-gb: Error: Invalid resource directory name
```

**Comment by misaochan** [comment]
@maskaravivek I think documentation was needed to address https://github.com/commons-app/apps-android-commons/pull/2947#pullrequestreview-237577334

**Comment by maskaravivek** [comment]
As discussed on Zulip, I will take up the documentation in another PR. :)

**Pull request merged**